devnotes/documentation.md

```diff
@@ -150,39 +149,33 @@ In order to build coreboot image, follow the steps below:
    Answer yes to the prompt and wait for the process to finish.
 
 4. Log into the BMC GUI again at https://\<BMC_IP\>/. Enter the Server power
-   operations (https://\<BMC_IP\>/#/server-control/power-operations) and invoke warm
-   reboot. Then move to Serial over LAN remote console
+   operations (https://\<BMC_IP\>/#/server-control/power-operations) and invoke
+   warm reboot. Then move to Serial over LAN remote console
    (https://\<BMC_IP\>/#/server-control/remote-console)
 
    Wait for a while until coreboot shows up:
 
-   ![](../images/cb_bootblock.png)
+   [![asciicast](https://asciinema.org/a/OTEPFRHlasyXQI2eRBLso0AB0.svg)](https://asciinema.org/a/OTEPFRHlasyXQI2eRBLso0AB0)
 
 5. Enjoy the coreboot running on Talos II.
 
 > **Optional:** In order to recovery the platform quickly to healthy state, flash
-> the HBB partition back with:\
+> the HBB partition back with: \
 > `pflash -e -P HBB -p /tmp/hbb.bin`
 
-## Running the coreboot on QEMU
-
-Please keep in mind, that QEMU doesn't implement many of the HW properties,
-that Talos II has. There may be many compatibility issues, or registers missing.
-
-More detailed informations can be found in [porting.md#qemu](porting.md#qemu)
-
-1. Clone the QEMU repository
-   ```
-   git clone git@github.com:qemu/qemu.git
-   # or HTTPS alternatively
-   git clone https://github.com/qemu/qemu.git
-   ```
-2. Build the QEMU ppc64 version
-   ```
-   cd qemu
-    ./configure --target-list=ppc64-softmmu && make
+## Memory configurations
+Following RAM configurations were tested and are proved to be properly initialized.
+   ```
+   MCS0, MCA0
+      DIMM0: 1Rx4 16GB
```

Comment:
Do we have some more details on the exact parts that were used?

Reply:
I have added information from the labels, but I don't know the details about about `1Rx8 8GB`

Reply:
I beleive the most important fact is that it is a **single** CPU setup and dual CPU setup is not supported currently

Reply:
I have added this information.